### PR TITLE
Add logs streaming for `botkube install` command

### PR DESCRIPTION
**Pull request by mszostok**
## Description

Changes proposed in this pull request:

- Add logs streaming for `botkube install` command

## Testing

1. Build the CLI:
    ```
    go build -o ./bin/botkube cmd/cli/main.go
    mv ./bin/botkube /usr/local/bin/botkube
    ```
3. Create k8s cluster: `k3d cluster create labs --image=rancher/k3s:v1.25.0-k3s1`
4. Run with valid data:

    ```bash
    export SLACK_CHANNEL_NAME=random
    export SLACK_APP_TOKEN="xapp-1-"
    export SLACK_BOT_TOKEN="xoxb-"
    
    botkube install --verbose \
    --set communications.default-group.socketSlack.enabled=true \
    --set communications.default-group.socketSlack.channels.default.name=${SLACK_CHANNEL_NAME} \
    --set communications.default-group.socketSlack.appToken=${SLACK_API_APP_TOKEN} \
    --set communications.default-group.socketSlack.botToken=${SLACK_API_BOT_TOKEN} \
    --set settings.clusterName=labs
    ```
4. Run with missing tokens:

    ```bash
    export SLACK_CHANNEL_NAME=random
    export SLACK_APP_TOKEN=""
    export SLACK_BOT_TOKEN=""
    
    botkube install --verbose \
    --set communications.default-group.socketSlack.enabled=true \
    --set communications.default-group.socketSlack.channels.default.name=${SLACK_CHANNEL_NAME} \
    --set communications.default-group.socketSlack.appToken=${SLACK_API_APP_TOKEN} \
    --set communications.default-group.socketSlack.botToken=${SLACK_API_BOT_TOKEN} \
    --set settings.clusterName=labs
    ```

6. Run with local repo and the most verbose option:
    ```bash
    botkube install --repo @local \
    --set analytics.disable=true \
    --set communications.default-group.socketSlack.enabled=true \
    --set communications.default-group.socketSlack.channels.default.name=${SLACK_CHANNEL_NAME} \
    --set communications.default-group.socketSlack.appToken=${SLACK_API_APP_TOKEN} \
    --set communications.default-group.socketSlack.botToken=${SLACK_API_BOT_TOKEN} \
    --set settings.clusterName=labs --verbose=trace
    ```
## Related issue(s)

Fix https://github.com/kubeshop/botkube/issues/1133

### CODE REVIEW

**Review by pkosiec — COMMENTED** [review]
Just a code review for now, looks great!

File: cmd/cli/cmd/install.go
Comment (pkosiec) [inline cmd/cli/cmd/install.go:1]: It would be great to introduce `uninstall` command 🤔

Comment (pkosiec) [inline cmd/cli/cmd/install.go:1]: When I ran a command in different dir, I got quite confusing error directly from Helm:

obraz

```
Error: while loading Helm chart: could not find protocol handler for:
```

Something to address in future maybe?

**Review by pkosiec — APPROVED** [review]
Awesome work! 🚀

File: cmd/cli/cmd/install.go
Comment (mszostok) [reply cmd/cli/cmd/install.go:1]: I will try to debug it 👍

Comment (mszostok) [reply cmd/cli/cmd/install.go:1]: I started to work on it 👍 will ping you with a dedicated PR 👍

**Pull request merged**